step02/api.yaml

```diff
@@ -0,0 +1,88 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: Thumbnail Service API
+  license:
+    name: MIT
+servers:
+  - url: http://lolcats.gg/v1/
+paths:
+  /users/me:
+    get:
+      tags:
+        - users
+      summary: Gets profile of current user
+      operationId: app.get_profile
+      responses:
+        '200':
+          description: User Profile
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
+        default:
+          $ref: '#/components/responses/GenericError'
+  /users/images/:
```

Comment:
can you make this `/images/presigned-url`? Since it's an image service API it should be namespaces in `images` rather than in the `user` API namespace

Reply:
oh...my fault, should have read that change closer.